### ISSUE TITLE
Corpus List Error Internal Server Error

### ISSUE THREAD

**Issue by dastevens05** [IBM/Train-Custom-Speech-Model#90]
Followed all steps, but receiving Corpus List Error
Internal Server Error when I navigate to Corpora.
Unable to upload the txt file.
Any ideas?

**Comment by dastevens05** [comment]
Actually I think the issue is I was on the Lite plan; moved to paid plan and now functions. May be worth adding to documentation that need a plan >Lite. Thank you.

**Comment by yhwang** [comment]
Yes, it's here: https://github.com/IBM/Train-Custom-Speech-Model#2-create-ibm-cloud-services
